iceoryx_posh/test/moduletests/test_capro_service.cpp

```diff
@@ -38,15 +38,22 @@ class ServiceDescription_test : public Test
     ServiceDescription csd1 = ServiceDescription(1, 2, 3);
```

Comment:
Please keep in mind that in a safety critical context the code and the test have to be maintainable by all developers.
This means. The tests require a understandable name. The name should state what you are doing and what the outcome is which you are expecting. 
Please study the `cxx::string` tests as an example: https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_utils/test/moduletests/test_cxx_string.cpp

Please refactor the unit tests.
Follow the AAA pattern: https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80
Introduction into MC/DC coverage: https://www.froglogic.com/coco/modified-condition-decision-coverage-mcdc

1. Every test should only test one single aspect. The reason is that with every test gtest constructs everything new from scratch so that a previous test case does not influence the outcome of a previous test case. If you test multiple aspects in a single test this can not be guaranteed.
2. Rename all the tests that they are having good and understandable names
3. Please do not create a huge amount of variables in advance with very short inexpressive names. Create them where you need them.